lib/chef/knife/node_policy_set.rb

```diff
@@ -0,0 +1,79 @@
+#
+# Author:: Piyush Awasthi (<piyush.awasthi@msystechnologies.com>)
+# Copyright:: Copyright 2017-2018, Piyush Awasthi
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "chef/knife"
+
+class Chef
+  class Knife
+    class NodePolicySet < Knife
+
+      deps do
+        require "chef/node"
+        require "chef/json_compat"
+      end
+
+      banner "knife node policy set NODE GROUP NAME (options)"
```

Comment:
Could this be `knife node policy set NODE POLICY_GROUP POLICY_NAME (options)`? That would be more clear as to what GROUP and the second NAME are.